docs/pages/contributing/documentation/how-to-contribute.mdx

```diff
@@ -1,168 +1,306 @@
 ---
-title: How to Contribute to Teleport's Documentation
-description: Follow this guide to get started contributing changes to Teleport's documentation
+title: How to contribute to Teleport documentation
+description: Follow this guide to learn how to contribute changes to Teleport documentation.
 ---
 
-This guide describes the general workflow for making documentation changes.
-It assumes you already have Node >=14 and `yarn` installed.
+We encourage everyone with an interest in improving Teleport documentation to contribute content, submit issues, and suggest changes.
+This guide describes the process for making changes to the documentation whether you are an internal or external contributor.
 
-## Step 1/4 Set up your local environment
+## Before you begin
 
-Clone the `gravitational/docs` repository via `git clone`:
+Before you begin, verify that your environment meets the following prerequisites:
 
-```code
-$ git clone https://github.com/gravitational/docs
-```
+* You have a code editor, a GitHub account, and experience using command-line programs, including `git` commands and command line options.
 
-Different versions of the documentation are organized into the `content`
-directory of `gravitational/docs`. Each subdirectory of `content` includes a git
-submodule for a different branch of `gravitational/teleport`.
+* You have Node.js, version 14.x or newer, installed. 
```

Comment:
```suggestion
* You have Node.js, version 16.x or newer, installed. 
```
We'll need to make this 18 in August, since Vercel is sunsetting 16.x support

Reply:
👍